### PR TITLE
Feature/dx 1234 correct typos dev portal

### PR DESCRIPTION
**Pull request by marinafrydenb**
Take a look at the payment-rousource.md as the code example is pink...

### CODE REVIEW

File: payment-instruments/invoice/seamless-view.md [hunk 1]
```diff
@@ -230,16 +230,16 @@ Lastly, initiate the Seamless View with a JavaScript call to open the `iframe`
 embedded on your website.
 
 {:.code-view-header}
-**JavaScript**
+**HTML**
 
-```js
-<script language="javascript">
+```html
+<html language="javascript">
```
Comment (asbjornu) [inline payment-instruments/invoice/seamless-view.md:236]:
```suggestion
<script language="javascript">
```

File: payment-instruments/invoice/seamless-view.md [hunk 2]
```diff
@@ -230,16 +230,16 @@ Lastly, initiate the Seamless View with a JavaScript call to open the `iframe`
...
     payex.hostedView.invoice({
         // The container specifies which id the script will look for to host the
         // iframe component.
         container: "swedbank-pay-seamless-view-page"
     }).open();
-</script>
+</html>
```
Comment (asbjornu) [inline payment-instruments/invoice/seamless-view.md:242]:
```suggestion
</script>
```

File: resources/public-migration-key.md [hunk 3]
```diff
@@ -21,59 +21,58 @@ paste it from below and save it to a file called `SwedbankPay.key`.
...
-nB3TOs/oC6xg/Aa0WlpOTrqoSvXLtrUKnWySmXgCBiUVKBaN1rfHYS4A39AXlL/H
-g1n1OiW4u9psgJk3GRBXXV2WE+J+qMuDUI1kwj+IbB/CDCXJ0ULXHHLY3QoSmAoy
-zRAJzq470jcrygaotgXvFmq3HJxXTquonxDtXXr7GqctsFVAeXvJM9JHZoOP/t3/
-lvZiiNQU42yfzOZJkNXoyPopsdWHynBUiAEy3GQdjwapgRIO2Jm6LDuWUI98n+mI
-5dXrTVGXKzoqzBFMdvJ8GBMMeGBW8WxjWF8eiqI5foMex8GS150aor930VIFlOk/
-PapmhjXEjvmR7fNuNqeD23CtSR+/vViD7VnCmpVtwvPiQlIFsPy94Se6IL0Rnxke
-iStodqloOXbxtYAzTEcdxZykm6NK6UTyB1kL0h4+2D32p7FRXdbYhXj6nUF1THaO
-X/EaNThP+8m3eoeiVIDYW3TPAfVwFfWFSVOc1HG3osSPZ9TkneBlzglhQKkvyTcq
-yxvnhIpS3k8lvKoUllx6H6TWndAD3aLXvv7wS3UugCuCzUzK9yAy0+YXNQARAQAB
+mQINBF46XsgBEACh1Yz2neopGj1UdGC+jDW+Cspq126X58EK1tWdo6mv3wpzN5GB
```
Comment (asbjornu) [inline resources/public-migration-key.md:26]: Why is the key changed?

**Review by asbjornu — CHANGES_REQUESTED** [review]

File: resources/public-migration-key.md (hunk 3 above)
Comment (marinafrydenb) [reply resources/public-migration-key.md]: Mistake! this is now corrected.

**Review by arebra — APPROVED** [review]
Great!

File: _includes/payment-resource.md [hunk 4]
```diff
@@ -34,7 +34,7 @@ Content-Type: application/json
 {:.code-view-header}
 **Response**
 
-```http
+```json
```
Comment (asbjornu) [inline _includes/payment-resource.md:37]:
```suggestion
```http
```

File: resources/public-migration-key.md [hunk 5]
```diff
@@ -21,7 +21,6 @@ paste it from below and save it to a file called `SwedbankPay.key`.
 {:.code-view-header}
 **Public Key**
 
-```pgp
```
Comment (asbjornu) [inline resources/public-migration-key.md:24]: No code fence? How does this look then?

**Review by asbjornu — CHANGES_REQUESTED** [review]

File: _includes/payment-resource.md (hunk 4 above)
Comment (marinafrydenb) [reply _includes/payment-resource.md]: fixed!

File: resources/public-migration-key.md (hunk 5 above)
Comment (marinafrydenb) [reply resources/public-migration-key.md:24]: Fixed!

**Review by asbjornu — APPROVED** [review]
🎉 ❤️

**Pull request merged**